packages/eslint-plugin/src/rules/prefer-output-readonly.ts

```diff
@@ -28,17 +28,26 @@ export default createESLintRule<Options, MessageIds>({
       [`PropertyDefinition:not([readonly=true]) > ${Selectors.OUTPUT_DECORATOR}`]({
         parent: { key },
       }: TSESTree.Decorator & { parent: TSESTree.PropertyDefinition }) {
-        context.report({
-          node: key,
-          messageId: 'preferOutputReadonly',
-          suggest: [
-            {
-              messageId: 'suggestAddReadonlyModifier',
-              fix: (fixer) => fixer.insertTextBefore(key, 'readonly '),
-            },
-          ],
-        });
+        report(key);
+      },
+      [`PropertyDefinition:not([readonly=true]):matches([typeAnnotation.typeAnnotation.typeName.name=OutputEmitterRef], [value.callee.name=output])`]({
```

Comment:
Would be nice to include the [`outputFromObservable`](https://angular.dev/api/core/rxjs-interop/outputFromObservable) function here as well, since it also creates an output:

```ts
class Test {
  testEmitter = outputFromObservable(someObservable$);
}
```
note that it returns an `OutputRef`, which is a super-type of `OutputEmitterRef`

```suggestion
      [`PropertyDefinition:not([readonly=true]):matches([typeAnnotation.typeAnnotation.typeName.name=/^Output(Emitter)?Ref$], [value.callee.name=/^output(FromObservable)?$/])`]({
```

Reply:
Dunno how readable those regexes are compared to the more standard `/^( | )$/` with alternatives, used e.g. here: https://github.com/angular-eslint/angular-eslint/blob/2f9e3784b193f475f40d514021c9844d10a745cd/packages/utils/src/eslint-plugin/selectors.ts#L2

Reply:
I ended up using a separate selector so that the message could use the correct type. i.e:

> Prefer to declare `@Output` as `readonly` since they are not supposed to be reassigned

vs

> Prefer to declare `OutputEmitterRef` as `readonly` since they are not supposed to be reassigned

vs

> Prefer to declare `OutputRef` as `readonly` since they are not supposed to be reassigned